Source/ButtonMerchant.swift

```diff
@@ -92,6 +92,26 @@ final public class ButtonMerchant: NSObject {
         core.trackIncomingURL(url)
     }
     
+    /**
+     Checks the URL in the passed NSUserActivity for a Button attribution and if present stores the token.
+     
+     - Attention:
+     To correctly attribute customers, you must call this method with every
+     incoming `userActivity` from the following `UIApplicationDelegate` method:
+     
+     - `application(_:userActivity:restorationHandler:)`
+     
+     - Parameters:
+     - activity: A NSUserActivity with which your app has been continued.
+     
+     */
+    public static func trackIncomingActivity(_ activity: NSUserActivity) {
```

Comment:
Lets go with `trackIncomingUserActivity` to stay consistent with Apple's terminology.